CustCartMethods.java

```diff
@@ -0,0 +1,141 @@
+import java.util.ArrayList;
+import java.util.List;
+/*
+ *Used to execute all the functions performed in a cart 
+ * 
+ */
+public class CustCartMethods
+{
+	public List<Product> cartvals=new ArrayList<Product>();
+	/**
+	 * Used to add items in the cart
+	 * @param productid represents product id
+	 * @param productquantity represents product quantity
+	 * @return whether the product was added or not
+	 */
+	boolean add(int productid,int productquantity) 

```

Comment:
use camelCase naming convention while declaring variable in the all the classes.
productid----> productId